docs/usage/configuration-options.md

```diff
@@ -676,6 +676,8 @@ Example:
 
 ## gitLabAutomerge
 
+Attention: This is a top-level key. To activate automerging for a specific `packageRule`, make sure to also set `automerge` to `true` in the specific rule!
```

Comment:
```suggestion
Attention: This is a top-level key.
To activate automerging within a `packageRule`, you must set `automerge` to `true` in that same `packagaRule`.
```

We use one sentence per line pattern.

Maybe this rewrite of the second sentence is clearer on what you need to do?

Reply:
Yeah, that sounds way better than mine!!